src/reset.scss

```
@@ -0,0 +1,7 @@
+*,
+::after,
+::before {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
```

You don't need reset margin, padding only with selector * 
```suggestion
* { 
  box-sizing: border-box;
}

::after,
::before {
  padding: 0;
  margin: 0;
}
```